extensions/adobe/experience/customerJourneyManagement/messageprofile.schema.json

```diff
@@ -89,6 +89,26 @@
             "twilio": "Twilio sms provider",
             "vibes": "Vibes sms Provider "
           }
+        },
+        "https://ns.adobe.com/experience/customerJourneyManagement/directMailChannelContext/routingDestination": {
+          "title": "Routing Destination",
+          "type": "string",
+          "description": "Routing Destination for Direct mail Channel , e.g. S3 or SFTP or Azue blob",
+          "meta:enum": {
+            "s3": "S3 routing Destination",
+            "sftp": "SFTP Routing Destination",
+            "azure": "Azure Routing Destination "
+          }
+        },
+        "https://ns.adobe.com/experience/customerJourneyManagement/directMailChannelContext/directMailFileName": {
+          "title": "Direct Mail FileName",
+          "type": "string",
+          "description": "Direct Mail FileName"
+        },
+        "https://ns.adobe.com/experience/customerJourneyManagement/directMailChannelContext/directMaildestinationId": {
```

Comment:
directMaildestinationId ==> directMaildestinationID

Reply:
Thanks @mjasrotia 
Updated

Reply:
@nikshar : It seems fields under directMailChannelContext are not event specific but action specific. I don't think we should be ingesting action level fields in user specific events.

We should review whether we can use entity dataset for ingesting these fields.

cc:  @chetanmeh